### ISSUE TITLE
Archive or delete this repository?

### ISSUE THREAD

**Issue by flowerornament** [urbit/doc#11]
Should we clean up old repositories to make Urbit in its entirety easier to grasp for newcomers?

**Comment by ohAitch** [comment]
As long as nobody makes a bunch of commits modernizing their archeological
status, they should get hidden by pagination from anyone not intentionally
looking for https://github.com/urbit/ repos specifically not updated in
years :P

On Wed, Apr 18, 2018 at 3:33 PM, Morgan Sutherland <notifications@github.com

**Comment by Fang-** [comment]
I agree with @msutherl on archiving/deleting explicitly. Looking at github.com/urbit, I'd rather see just things that are actually useful (be that as reference or as code) rather than the current list of repositories that haven't been touched in years.

Of course you can see they haven't been touched in years, but if we can reduce clutter for our users than we should always do so.

**Comment by ohAitch** [comment]
when I look at github.com/urbit, the first page does not contain anything
before Jan 12 of this year, but I suppose some of those are private and
don't normally show up. I think having "old" or "archeo" in the title is
plenty explicit, but if you want to create a separate urbit-archeology org
or sth and renamespace things there, I wouldn't hugely object.

Big -1 on deleting things outright however, I am *currently in the process
of* sending Rob an email referencing urbit/old-urbit.org documentation that
we want to  cross-reference with information currently on the site.

**Comment by Fang-** [comment]
Oh yeah hard deleting is not a good idea. A github.com/urbit-archeology sounds nice!

**Comment by yebyen** [comment]
So the idea is that you move the content of this repo to urbit-archaeology and then empty out this repo, and now everything is there, but keep a reference to it here?  Then everyone's links are broken...

I'm not sure what exactly that buys us.  It does say "old-doc" right there in the repo name, so I don't think it's likely to be confused for something current or updated.
